LDEventSource/LDEventSource.m

```diff
@@ -72,7 +72,7 @@ - (instancetype)initWithURL:(NSURL *)URL httpHeaders:(NSDictionary<NSString*, NS
         _retryInterval = ES_RETRY_INTERVAL;
         _retryAttempt = 0;
         _httpRequestHeaders = headers;
-        messageQueue = dispatch_queue_create("co.cwbrn.eventsource-queue", DISPATCH_QUEUE_SERIAL);
+        messageQueue = dispatch_queue_create("co.cwbrn.ldeventsource-queue", DISPATCH_QUEUE_SERIAL);
```

Comment:
If we're changing this, it might make sense to change the prefix to `com.launchdarkly`.